CONTRIBUTING.md

```diff
@@ -63,12 +124,96 @@ $ npm test -- --config jest.config.json
 # etc
 ```
 
-### Run coverage
+#### Run coverage
+
+As the Boy Scout Rule states, you are expected to leave the code better than you found it. To do so, please check that your code is properly covered by meaningful tests.
 
 ```sh
 npm test -- --coverage
 ```
 
+#### Browser testing
+
+You can find the list of supported browser in the [List of supported browsers](./README.md#list-of-supported-browsers) section of the README.
+
+If you are developping on a Mac, you will find some Virtual Machines to help you test, on [Microsoft Edge Tools](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/) webiste.
+
+### Documentation
+
+After working on a component, you are expected to update the documentation.
+
+Documentation can be found in the [Design System](https://github.com/AxaGuilDEv/design-system) repository.
+
+Do not forget to update the component status page, located in this repository at:
+
+```text
+└─ src/
+    └─ pages/
+      └─ get-started/
+          └─ component-status/
+            └─ markdown/
+                └─ content.md
```

Comment:
We can maybe add a link?

Reply:
True, that makes sense now that it's in a different repo (though there is a link on line 145)

NB: link will be on master, is that OK?